cmd/kueuectl/app/util/client_getter.go

```diff
@@ -2,14 +2,16 @@ package util
 
 import (
 	"k8s.io/cli-runtime/pkg/genericclioptions"
-
+	k8s "k8s.io/client-go/kubernetes"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/oidc"
```

Comment:
```
_ "k8s.io/client-go/plugin/pkg/client/auth/oidc"
```
FYI -- oidc provider needs to imported for connecting clusters with OIDC enabled from `kueuectl`
If its not imported it will throw the below error:
```
Error: no Auth Provider found for name "oidc"
```

Reply:
Yes, please, create separate PR.

Reply:
And also I think it should be:

```
import _ "k8s.io/client-go/plugin/pkg/client/auth"
```

Reply:
Could you please create PR for this? We already added for kjobctl on #2515. And it would be great to add it to v0.8.0.

Reply:
Thanks. Done https://github.com/kubernetes-sigs/kueue/pull/2602